qiskit/transpiler/passes/commutation_analysis.py

```diff
@@ -73,7 +73,12 @@ def run(self, dag):
 
                 if current_gate not in current_comm_set[-1]:
                     prev_gate = current_comm_set[-1][-1]
-                    if _commute(current_gate, prev_gate):
+                    does_commute = False
```

Comment:
A simplification:
```python
try:
    if _commute(current_gate, prev_gate):
        current_comm_set[-1].append(current_gate)
except TranspilerError:
    pass
```

Reply:
not sure if this is the same as the `else` falls outside the try/except.

Reply:
You are right, the blank line confused me.. let's leave it as it is.